patient-appointment-api.yml

```diff
@@ -101,16 +101,16 @@ paths:
       tags:
         - Scheduled Appointments
       summary: Patient Data Processing
-      description: "# Data Requirements\n\n**Populate all known data**. It is required\
+      description: "\n# Data Requirements\n\n**Populate all known data**. It is required\
         \ for processing that as much information as possible is supplied.\n \nAll\
         \ active diagnoses and medications for a patient should be supplied in every\
         \ API request. \nThat information is used to satisfy reciprocity requirements\
         \ of the various health information exchanges \nthat Connective Health participates\
-        \ in. Not populating known diagnoses and medications for each patient would\
-        \ \nbe a violation of health information exchange reciprocity rules which\
-        \ could result in the termination of information exchange.\n\n## Overview\n\
-        Standard processing for clinical patient information, both summarized and\
-        \ complete information is supported."
+        \ in. Failure to populate known diagnoses and medications for a patient is\
+        \ \na violation of health information exchange reciprocity rules, and may\
```

Comment:
guessing these internal newlines are also necessary / coming from OpenAPI

Reply:
This is just for readability within the code as well. Otherwise it's just a single giant line.